CMakeLists.txt

```diff
@@ -337,14 +337,18 @@ if(LOVR_USE_PICO)
 endif()
 
 if(LOVR_BUILD_SHARED)
+  list(REMOVE_ITEM LOVR_SRC src/main.c)
```

Comment:
On Android LÖVR is built as a shared library and `os_android.c` calls `main`, requiring this file to be compiled.  Am I missing something?

Reply:
If desktop platforms need main omitted (which seems sensible), then seems like `LOVR_BUILD_SHARED` is not flexible enough to express "should compile `main`".  Android should either not use/respect `LOVR_BUILD_SHARED` or there needs to be a separate `LOVR_NO_MAIN` for desktop platforms to use.

Reply:
ugh you're right. how did this work for me? It's not so much desktop platforms, but rather standalone desktop apps that use lovr as a library and have their own main. having a main in a shared library is really weird imho but if that's what android requires, that's that. I could define a LOVR_NO_MAIN, that's an okay workaround...

Reply:
I've added a commit with a proposed solution. I used LOVR_USE_MAIN instead of NO_MAIN to avoid double negation. It's not super pretty, and I'm inferring some stuff. If you don't like it I can also make it an always-on option and just override it from alloverse's cmake. lmk what you think.